flare/gp.py

```diff
@@ -962,3 +933,76 @@ def __del__(self):
         if (self.name in _global_training_labels):
             _global_training_labels.pop(self.name, None)
             _global_training_data.pop(self.name, None)
+
+    @staticmethod
+    def backward_arguments(kwargs, new_args={}):
```

Comment:
Love these deprecation warnings. It's great we have this! Will save a lot of grief in using old scripts